### ISSUE TITLE
Atari 2600 C support

### ISSUE THREAD

**Issue by gus0650** [sehugg/8bitworkshop#175]
is it feasible? cc65 should support it.

**Comment by sehugg** [comment]
Now that I have ca65 (the assembler) working with the VCS, it wouldn't be hard to activate cc65. You would have to code mostly in assembler, it is too slow for things like display kernels. It'd be an interesting experiment though.

**Comment by gus0650** [comment]
i can code in C but not assembler. i am a bit surprised that C is not fast enough, as there is even a BASIC. i suppose it would require a high level C API to make it work? 

by the way there is also a compile target for the Atari 7800 for CC65, so perhaps that is an alternative. the 7800 is more powerful.

**Comment by mariogazziro** [comment]
i have ported an integer mandelbrot algorithm to C in 8bitworkshop NES 6502 and it runs as fast as the original assembler version,.. so i dont think that C is not fast enough and since 6502 wasnt much better than atari´s 6507 processor, may be your C code was not completly optmized. 
C version:
https://github.com/mariogazziro/mandelbrot
original asm:
https://www.youtube.com/watch?v=Fui-NoI-qfQ

**Comment by sehugg** [comment]
I've added the atari2600 CC65 target to the dev build. I got the Hello World working: https://github.com/cc65/cc65/blob/master/samples/atari2600/hello.c

I haven't yet found very much sample code against this target, but it should be fun to try some things. There is no high-level support library, just TIA and RIOT header files.

CA65 works with the VCS platform if you use the ".ca65" extension, it requires a bit of work to convert DASM code  though.
